airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -330,10 +335,6 @@ WITH intermediate_data AS (
               FROM ${raw_table_id}
               WHERE
                 _airbyte_loaded_at IS NULL
-                AND (
```

Comment:
nit: delete the comment above (`// Note that we intentionally excluded deleted records from this insert`)